tests/functional/curate/cram/titlecase.t

```diff
@@ -3,16 +3,44 @@ Setup
   $ pushd "$TESTDIR" > /dev/null
   $ export AUGUR="${AUGUR:-../../../../bin/augur}"
 
+Test output with lowercase and already captialized strings.
 
-Create NDJSON file for testing titlecase with different forms
+  $ echo '{"record": 1, "authors": "john smith", "author2": "Jane Doe"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "authors" "author2"
+  {"record": 1, "authors": "John Smith", "author2": "Jane Doe"}
 
-  $ cat >$TMP/records.ndjson <<~~
-  > {"record": 1, "authors": "john smith", "author2": "Jane Doe"}
-  > ~~
+Test output with lowercase articles.
 
+  $ echo '{"record": 2, "title": "the night OF THE LIVING DEAD", "location": "BRAINE-LE-COMTE, FRANCE"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title" "location" --articles "a" "and" "of" "the" "le"
+  {"record": 2, "title": "The Night of the Living Dead", "location": "Braine-le-Comte, France"}
 
-Test output with Unicode normalization form "NFKC".
+Test output with abbreviations
```

Comment:
Ditto with this copy/paste typo fix.

Reply:
fixed in https://github.com/nextstrain/augur/commit/c00156ba7d31026aa89fc5ed68ae5215c09b99bd